src/JabbR-Core/Properties/Chat.nuget.css

```diff
@@ -0,0 +1,131 @@
+﻿.nuget-badge {
```

Comment:
This was all previously under content/themes/default, lets keep out of properties (how did it end up here?)